decentraland/unity-explorer
# Fix: scene unloading cancelation

## What does this PR change?
fix #460
handled issue when ISceneFacade was finishing loading after world was disposed.

## How to test the changes?

1. Try simple scenario of jumping between realm, just to verify that it is not presenting new bugs (compare with current `main`).

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
